### ISSUE TITLE
Custom aspect ratio

### ISSUE THREAD

**Issue by prohtex** [lhc70000/iina#1341]
Sometimes, I watch videos that are encoded at the wrong aspect ratio. I'd like to request a feature that would allow the user to dynamically alter the aspect ratio by resizing the window. One way to do this would be holding down "shift". This way, I could hold "shift" and drag to resize the window, and the video would change aspect to match the shape of the window.

I know there are Aspect Ratio settings, but sometimes I play videos that do not conform to any of these predefined sizes, and in IINA there doesn't seem to be a way to set a custom aspect ratio.

Thanks!

**Comment by saagarjha** [comment]
There should be a text field under the "Aspect Ratio" section the Quick Video Settings that allows you to set a custom aspect ratio. Is this what you wanted?

**Comment by prohtex** [comment]
Yes, that gets the job done. If possible I would like to leave this request open, in case other users are interested in being able to drag with a keyboard modifier to enable dynamic scaling of the video. I think this would be a really great feature, as it would allow the user to very rapidly change the aspect ratio without having to do any math.

The problem I have is that some of the video files I use are encoded to an aspect ratio that is only slightly off from the source. I know that Aspect Ratio is typically used to correct issues caused by anamorphic videos, which have a "correct" ratio that the user usually knows. But some videos come from sources that are not "correct." In these cases, it would be very helpful to be able to drag to make quick minor adjustments.
